test/testmetrics/metrics/04-teststep-retrieve-metrics-via-localhost.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1
```

Comment:
Please rename this file to something more descriptive, its functionality is not to retrieve metrics via localhost, but to check the active metrics